commons/dev/configmaps/m2m-event-migrations-configmap.yaml

```
@@ -0,0 +1,184 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: m2m-event-migrations
```

```suggestion
  name: flyway-m2m-events
```
consistency with other migrations